models/template.go

```diff
@@ -14,6 +15,7 @@ type Template struct {
 	Id           int64        `json:"id" gorm:"column:id; primary_key:yes"`
 	UserId       int64        `json:"-" gorm:"column:user_id"`
 	Name         string       `json:"name"`
+	EnvelopeSender      string       `json:"envelope_sender"`
```

Comment:
If possible, could you `gofmt` the code in this pull request? This ensures consistent formatting across Gophish and is the recommended way to format Golang.

The easiest way to do this is to configure your editor to run a tool like `goimports` on every save.